patches/minecraft/net/minecraft/entity/Entity.java.patch

```diff
@@ -310,9 +316,9 @@
 +      this.reviveCaps();
 +   }
 +
-+   private float getEyeHeightForge(Pose pose, EntitySize size) {
-+      net.minecraftforge.event.entity.EntityEvent.EyeHeight evt = new net.minecraftforge.event.entity.EntityEvent.EyeHeight(this, pose, size, this.func_213316_a(pose, size));
++   private net.minecraftforge.event.entity.EntityEvent.Size getSizeForge(Pose pose, EntitySize size) {
```

Comment:
honestly all of this should move to ForgeHooks/ForgeEvenntFadctory like all the other events instead of being special.

Reply:
moved